lesson02.md

```diff
@@ -0,0 +1,440 @@
+<!-- .element: class="fragment" -->
+
+---
+
+### Variables are variable
+
+You can change the value of a variable using the assignment operator (`=`):
+<!-- .element: class="fragment" -->
+
+```js
+let temperature = 24;   // define a variable, initialize it to 24
+temperature = 30;       // assign a new value (30) to the variable
+temperature = 23;       // now temperature is 23
+```
+<!-- .element: class="fragment" -->
+
+We define a variable once, then we can change it as often as needed.
+<!-- .element: class="fragment" -->
+
+---
+
+### try it!
+
+Create some variables:
+
+* one for your full name
+* one for your age
+* one for your favorite city
+* one for whether you can speak German or not
+* and anything else you can think of :)
+
+In the JavaScript console, enter the name of the variable. Do you see its value?
+
+Bonus: Change the value of one of the variables that you defined.
+
+---
+
+### Variables
+
+Variables can be used wherever we can use values
+
+```js
+let pricePerTicket = 8;
+let friends = 3;
+let totalPrice = friends * pricePerTicket;
+```
+<!-- .element: class="fragment" -->
+
+---
+
+### Why do we use variables?
+
+We use variables
+
+* to give names (meaning) to values. "42" could mean a person’s age or shoe size.
+<!-- .element: class="fragment" -->
+* for reuse and flexibility. We define the variable once with a value, and use it often.
+<!-- .element: class="fragment" -->
+
+---
+
+### Variable names
+
+* You can use letters, numbers, and underscore `_` (spaces are *not* allowed!)
+* Variable name cannot start with a number.
+
+Valid variable names:
+```js
+let element = 3;
+let element3 = 5;
+```
+
+Invalid variable names:
+```js
+let 2squared = 4;
+let element-1 = 0;
+let full name = 'Anakin Skywalker';
+```
+
+---
+
+Giving good and descriptive names to your variables is very important!
+
+Good variable names make the code easier to understand by other developers, and even by yourself!
+
+Valid but not very descriptive variable names:
+
+```js
+let a = 0;
+let _12 = 0;
+let asldjf = 0;
+let thisisareallylongvariablename = 0;
+```
+
+---
+
+In this course, and JavaScript in general, we use 'camelCase':
+
+- `isStudent`
+- `favoriteBook`
+- `likesGermanFood`
+
+---
+
+Variable names are case sensitive:
+
+```js
+let name = 'John';
+let Name = 'John';
+let NAME = 'John';
+```
+
+These are 3 different variables.
+
+---
+
+### Practice
+
+Let's say we want to go to the cinema with some friends (choose any number).
+
+How many people are going to the cinema in total? Create a variable for that.
+<!-- .element: class="fragment" -->
+
+A ticket to watch the movie costs 12.5€, create a variable for this as well.
+<!-- .element: class="fragment" -->
+
+Create a new variable that contains how much we have to pay in total.
+<!-- .element: class="fragment" -->
+
+
+Ada and Alan volunteered to pay for the tickets. Use JavaScript to compute how much each has to pay.
+<!-- .element: class="fragment" -->
+
+---
+
+### Bonus
```

Comment:
I understand we wanted to name these Homework instead? As to clarify that they are tasks which are to be taken home.

Reply:
It's a Tuesday's class, so we traditionally don't give homeworl

Reply:
got it, makes sense